apps/web/src/app/[locale]/_components/feature-card.tsx

```diff
@@ -402,7 +402,7 @@ export function CollaborativeEnvironmentCard(props: CardProps) {
         >
           <div className="flex items-center gap-2">
             <Badge name="dax" />
-            <div className="text-xs text-neutral-500">5 years ago</div>
```

Comment:
me thinks 5 years ago was more funnier

Reply:
yeah but it suggests an inactive community